### PR TITLE
support for include and pragma include shader macros

### PR DESCRIPTION
**Pull request by patriciogonzalezvivo**
Adding support for both 

```glsl
#include "dependency.glsl"
```

```glsl
#pragma include "dependency.glsl"
```

to `ofShaders`

### CODE REVIEW

File: libs/openFrameworks/gl/ofShader.cpp [hunk 1]
```diff
@@ -414,6 +414,8 @@ bool ofShader::setupShaderFromSource(ofShader::Source && source){
  * https://www.opengl.org/discussion_boards/showthread.php/169209-include-in-glsl?p=1192415&viewfull=1#post1192415
  */
 
+typedef std::vector<std::string> StringList;
+
 //--------------------------------------------------------------
```
Comment (ofTheo) [inline libs/openFrameworks/gl/ofShader.cpp:419]: Can we remove the typedef? 
Then I think it will be good to go! 🙂

Comment (patriciogonzalezvivo) [reply libs/openFrameworks/gl/ofShader.cpp]: Ups! Good catch!

**Comment by ofTheo** [comment]
Thanks @patriciogonzalezvivo!  🙏

**Comment by patriciogonzalezvivo** [comment]
Thank you guys!! This is going to be FUN!

**Pull request merged**